assets/style.css

```diff
@@ -65,6 +69,10 @@ a:hover {
     text-decoration: underline;
 }
 
+img {
+    width: 100%;
+}
+
```

Comment:
Please dont make changes on the templates like these, since it will affect all other lectures.